content/direct-deposit-management/release-notes/2024-03-20.md

```diff
@@ -0,0 +1,7 @@
+We added Education Benefits payments to the Direct Deposit Management API so that all VA payments are associated to one bank account. This offers streamlined convenience for payees as well as decreases fraud opportunities.
```

Comment:
```suggestion
We added Education Benefits payments to the Direct Deposit Management API so that all VA payments are associated with one bank account. This offers streamlined convenience for payees and decreases fraud opportunities.
```